### PR TITLE
Scan: Support fast sync

### PR DESCRIPTION
**Pull request by evilpeach**
## Implementation details
1. Display "Syncing" when `txHash`, `txTimestamp`, `requester` is None (undefined)
   1. Oracle Script index page(request tab)
   2. Data source index page(request tab)
image
   3. Request Index Page
image
   4. Validator Index Page (Report Tab)
2. Disable Copy Non-EVM Proof (It will solve later)
3. Display block hasn't mined yet and block is syncing on Block Index Page
image
image
4. Display transaction hash is syncing or not found.
image
image
5. Remove unused function on AxiosRequest
6. Show Syncing Message Modal
image
 

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [x] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [x] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: scan/src/components/data-source/DataSourceRequestTable.re [hunk 1]
```diff
@@ -63,12 +63,16 @@ let renderBody = (reserveIndex, requestsSub: ApolloHooks.Subscription.variant(Re
...
-             <Timestamp.Grid
-               time=txTimestamp
-               size=Text.Md
-               weight=Text.Regular
-               textAlign=Text.Right
-             />
+             switch (txTimestamp) {
+             | Some(txTimestamp') =>
+               <Timestamp.Grid
+                 time=txTimestamp'
```
Comment (Pitchanai) [inline scan/src/components/data-source/DataSourceRequestTable.re:69]: Is this better ?
```
| Some({time: txTimestamp}) =>
```

**Review by pzshine — APPROVED** [review]

**Review by pzshine — CHANGES_REQUESTED** [review]
1. Balance didn't decrease when we sent BAND
image

2. Owasm code is disappeared
image

File: scan/src/pages/BlockIndexPage.re [hunk 2]
```diff
@@ -15,6 +15,24 @@ module Styles = {
...
+
+  let pageContainer =
+    style([
+      width(`percent(100.)),
+      paddingTop(`px(50)),
+      minHeight(`px(450)),
+      display(`flex),
+      flexDirection(`column),
+      alignItems(`center),
+      justifyContent(`center),
```
Comment (Pitchanai) [inline scan/src/pages/BlockIndexPage.re]: Please use `CssHelper.flexBox()` instead.

**Review by Pitchanai — CHANGES_REQUESTED** [review]

File: scan/src/components/data-source/DataSourceRequestTable.re (hunk 1 above)
Comment (evilpeach) [reply scan/src/components/data-source/DataSourceRequestTable.re:69]: It isn't object.

**Comment by evilpeach** [comment]
> 1. Balance didn't decrease when we sent BAND
>    image
> 2. Owasm code is disappeared
>    image

Answer
1. Server was delay.
2. Some oracle script doesn't have sourceCodeURL.

**Review by pzshine — CHANGES_REQUESTED** [review]
Typo

it should be:

`Please wait until the state is up to date`

**Review by pzshine — APPROVED** [review]

**Review by Pitchanai — APPROVED** [review]

**Pull request merged**